contrib/pacman/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+pkgname='gollama'
+pkgdesc='Go manage your Ollama models'
+pkgver='1.27.6'
```

Comment:
How does this version keep in sync with the git tags?

Reply:
It doesn't, however the newly committed versions do automatically update the version in addition to skipping the checksum verification. `git describe` is used to with `sed` to extract the version number based on the tag. See [PKGBUILD#L17-L20](https://github.com/sammcj/gollama/blob/0ce3863e8360839ad56a07914262c3a18847ed0a/contrib/pacman/gollama-git/PKGBUILD#L17-L20)